lizard_auth_client/views.py

```diff
@@ -844,8 +877,33 @@ def form_valid(self, form):
         # portal that initiated the creation of the new SSO account. This may
         # vary since the server can have multiple hostnames.
         payload['visit_url'] = self.request.get_host()
-        response = _sso_post('new-user', payload)
-        updated_values = response['user']
+        try:
+            json_data = _new_user_sso_post('new-user', payload)
+        except requests.exceptions.HTTPError as e:
+            json_data = e.response.json()
```

Comment:
Huh? Als er een error is ("server down") is er toch vaak geen nette json data? Dan gaat er hier iets mis in de error handling.

En de "except" is wel erg groot. 1 regel in de try, 20 onder de except?